### PR TITLE
PIL-692 Feature/Remove BTC access

### PR DESCRIPTION
**Pull request by jdnmr**
This PR aims to completely kill user access to any BTC-related features:

- Remove `Bitcoin wallet` from `Accounts` selection;
- Remove `BTCView` from `Assets` screen;
- Filter out BTC events on Home screen's Activity Feed;
- On first app launch, switch blockchain and account to Ethereum/SW or KW (refers to those users who had `Bitcoin wallet` active during last app usage);
- Delete some redundant BTC-related utils, actions and local methods where it was safe to do. 

Note that this PR doesn't entirely remove BTC-related code, nor does it delete BTC stuff from local storage. These changes will be applied in future releases. For now we just want to disable any form of BTC usage and information displaying.

### CODE REVIEW

File: src/actions/accountsActions.js [hunk 1]
```diff
@@ -297,3 +298,15 @@ export const initOnLoginSmartWalletAccountAction = (privateKey: string) => {
     dispatch(setUserEnsIfEmptyAction());
   };
 };
+
+export const fallbackToSmartOrKeyAccountAction = () => {
+  return async (dispatch: Dispatch, getState: GetState) => {
```
[the remark this answers is not in the document]
Comment (jsidorenko) [reply src/actions/accountsActions.js:305]: It's more a Redux specific pattern. You can obviously avoid putting async here, but once you add await it's very common you will forget to put the async back :)

File: src/utils/accounts.js [hunk 2]
```diff
@@ -166,3 +166,9 @@ export const getSmartWalletAddress = (accounts: Accounts): ?string => {
   if (!swAccount) return null;
   return getAccountAddress(swAccount);
 };
+
+const getSupportedAccountTypes = () => Object.values(ACCOUNT_TYPES);
+
+export const isSupportedAccountType = (accountType: string) => {
+  return getSupportedAccountTypes().includes(accountType);
+};
```
[the remark this answers is not in the document]
Comment (jsidorenko) [reply src/utils/accounts.js:174]: ^^ it's harder to read, imho

File: src/actions/accountsActions.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsidorenko) [reply src/actions/accountsActions.js:305]: and welcome long nights of debugging :D

[the remark this answers is not in the document]
Comment (IAmKio) [reply src/actions/accountsActions.js:305]: Gotcha - thanks for the explanation! I'd seen it too many times to think it must be a pattern of sorts.

[the remark this answers is not in the document]
Comment (jdnmr) [reply src/actions/accountsActions.js:305]: You're right about the `async/await`, I guess we all just copy-paste without thinking much 😄. Changed this one. As for the action returning a function, we've been using this pattern literally everywhere throughout the app, so I would leave it this way to keep consistent.

File: src/utils/accounts.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jdnmr) [reply src/utils/accounts.js]: 👍

**Pull request merged**